src/scmdata/run.py

```diff
@@ -1569,6 +1590,16 @@ def process_over(
             If the operation is not an allowed operation
 
             If the value of na_override clashes with any existing metadata
+
+            If ``operation`` produces a :class:`pandas.Series`, but `as_run`` is True
+
+            If an invalid value for ``as_run`` is provided
+
+        :class:`scmdata.errors.MissingRequiredColumnError`
+            If `as_run` is not False and the result does not have the required metadata
+            to convert to an :class`ScmRun <scmdata.ScmRun>`.
+            This can be resolved by specifying additional metadata via ``op_cols``
+
```

Comment:
Add documentation for raising `ValueError` if `as_run` is not a subclass of :class:`scmdata.BaseScmRun`?